packages/govern/README.md

```diff
@@ -0,0 +1,32 @@
+# govern.js
+
+> The JavaScript client API for the Govern server
+
+## Usage
+
+**Usage with default config**
+
+> The default config uses the Govern servers deployed by Aragon
+
+``` javascript
+import { dao } from '@aragon/govern'
+
+const response = await dao('0x0...');
```

Comment:
I’m thinking now that having the a functional API will make it impossible to do `const dao = …` since `dao` is already taken by the function name. But maybe it’s not that bad, since it will force people to name what they fetch? e.g. `const myDao = await dao()`

Reply:
This isn't that bad.. they have also the possibility to rename it with the import :)